README.md

```diff
@@ -146,12 +146,13 @@ Fluent Bit is fully supported on Windows environments, get started with [these i
 | [influxdb](https://docs.fluentbit.io/manual/pipeline/outputs/influxdb) | InfluxDB | Flush records to InfluxDB time series database. |
 | [kafka](https://docs.fluentbit.io/manual/pipeline/outputs/kafka) | Apache Kafka | Flush records to Apache Kafka |
 | [kafka-rest](https://docs.fluentbit.io/manual/pipeline/outputs/kafka-rest-proxy) | Kafka REST Proxy | Flush records to a Kafka REST Proxy server. |
-| [nats](https://docs.fluentbit.io/manual/pipeline/outputs/nats) | NATS | flush records to a NATS server. |
-| [null](https://docs.fluentbit.io/manual/pipeline/outputs/null) | NULL | throw away events. |
+| [nats](https://docs.fluentbit.io/manual/pipeline/outputs/nats) | NATS | Flush records to a NATS server. |
+| [null](https://docs.fluentbit.io/manual/pipeline/outputs/null) | NULL | Throw away events. |
+| [s3](https://docs.fluentbit.io/manual/pipeline/outputs/s3) | NULL | Flush records to s3 |
```

Comment:
Title should be S3 not Null?

Reply:
The title is S3. The null you're looking at is on the above line. The above line shows as changed because I capitalized "throw"

Reply:
<img width="530" alt="Screen Shot 2020-11-11 at 2 56 27 PM" src="https://user-images.githubusercontent.com/29443996/98873937-1dae7480-242e-11eb-8de8-db6cac85e239.png">

I see the title as NULL

Reply:
Oh, I was looking at the name, my bad.